api/lib/infrastructure/repositories/complementary-certification-repository.js

```diff
@@ -32,14 +32,23 @@ const getTargetProfileById = async function ({ complementaryCertificationId }) {
     .orderBy('complementary-certification-badges.createdAt', 'desc')
     .first();
 
+  const currentTargetProfileBadges = await knex('badges')
+    .select({
+      id: 'badges.id',
+      label: 'complementary-certification-badges.label',
+      level: 'complementary-certification-badges.level',
+    })
+    .leftJoin('complementary-certification-badges', 'complementary-certification-badges.badgeId', 'badges.id')
+    .where({ targetProfileId: currentProfile.id });
+
   const complementaryCertification = await knex
     .from('complementary-certifications')
     .where({ id: complementaryCertificationId })
     .first();
 
   return new ComplementaryCertificationForAdmin({
     ...complementaryCertification,
-    currentTargetProfile: { ...targetProfile },
+    currentTargetProfile: { ...currentProfile, badges: [...currentTargetProfileBadges] },
```

Comment:
Est-ce que la destructuration est necessaire?
```suggestion
    currentTargetProfile: { ...currentProfile, badges: currentTargetProfileBadges },
```